### PR TITLE
Add -worker suffix to the worker NEW_RELIC_APP_NAME

### PR DESCRIPTION
**Pull request by bladealslayer**

### CODE REVIEW

File: script/startup.sh [hunk 1]
```diff
@@ -25,6 +25,8 @@ case "$APP_MODE" in
             # Do nothing
             exec sleep 86400
         else
+            # add -worker suffix to newrelic app name to separate
```
Comment (lyyder) [inline script/startup.sh]: Separate what from what?

**Comment by lyyder** [comment]
Didn't understand the comment, otherwise 👍

File: script/startup.sh (hunk 1 above)
Comment (bladealslayer) [inline script/startup.sh]: Thanks, will fix!

**Comment by bladealslayer** [comment]
I think this is not really needed. Closing.

**Pull request closed without merging**